bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -46,6 +46,28 @@ spec:
             source "/env/${K8S_NODE}"
             set +o allexport
           fi
+
+          # Wait for ovn-monitor-all=true before ovn-controller starts to
+          # avoid a race condition in ovn-controller:
+          # https://bugzilla.redhat.com/show_bug.cgi?id=1808125
+          trap 'kill $(jobs -p); exit 0' TERM
+          retries=0
+          while true; do
+            monitor_all=$(ovs-vsctl --columns _uuid find Open_vSwitch external_ids:ovn-monitor-all="true" 2> /dev/null | wc -l)
+            if [[ "${monitor_all}" -eq 0 ]]; then
+              echo "warning: OVS external_ids:ovn-monitor-all not set yet, waiting 5s ..." 2>&1
+              sleep 5 & wait
```

Comment:
@dceara let's reduce the wait from 5 seconds to 1 or something like that, and increase the retries to 2 minutes.

Reply:
@dcbw Done. I'm also unsure about the e2e-gcp-ovn test failures. Scheduling a rerun.

/retest